core/store/src/trie/shard_tries.rs

```diff
@@ -393,19 +426,11 @@ impl ShardTries {
                     || trie_config.load_mem_tries_for_shards.contains(shard_uid)
```

Comment:
Should you also check the new config and the tracked shards?

Reply:
This function is called upon startup only.
It uses `load_mem_tries_for_all_shards` and `load_mem_tries_for_shards` to specify for which shards we load memtrie at the beginning.
With stateless validation, we will likely remove this functionality.
If `load_mem_trie_for_tracked_shards` is enabled, tracked shards will be loaded on its own way.

Reply:
Don't we need to load the memtries for tracked shards at startup? Or does that happens elsewhere?

Reply:
We need to run state sync for tracked shards at startup, so they will be loaded later during state sync.